cypress.json

```diff
@@ -2,5 +2,16 @@
   "video": false,
   "screenshotOnRunFailure": false,
   "defaultCommandTimeout": 20000,
-  "responseTimeout": 20000
-}
+  "responseTimeout": 20000,
+  "reporter": "cypress-multi-reporters",
+  "reporterOptions": {
+    "reporterEnabled": "mochawesome",
+    "mochawesomeReporterOptions": {
+      "reportDir": "cypress/reports/mocha",
+      "quite": true,
+      "overwrite": false,
+      "html": true,
+      "json": true
+    }
+  }
+}
```

Comment:
EOF

Reply:
Done... reinstalled extension to format code and fixes were achieved.